### PR TITLE
[enhancement] pr-backport.sh - make it more understandable when using gh and git

### PR DESCRIPTION
**Pull request by lhanusov**
Signed-off-by: Lukas Hanusovsky <lhanusov@redhat.com>

### CODE REVIEW

**Review by jonkoops — DISMISSED** [review]

File: .github/scripts/pr-backport.sh [hunk 1]
```diff
@@ -83,3 +83,6 @@ git push origin $PR_BRANCH:$PR_BRANCH --set-upstream
 
 echo_header "Opening web browser to create pull request"
 gh pr create -B $TARGET_BRANCH -f -w
+
+echo_header "Checkout to 'main' branch"
+git checkout main
```
Comment (mhajas) [inline .github/scripts/pr-backport.sh:88]: I would not check out the main branch. My local `main` is quite outdated and can cause much pain for IntelliJ to reindex the old branch.

What if we check out to a previous branch by storing the result of `git rev-parse --abbrev-ref HEAD` to some variable?

**Review by mhajas — COMMENTED** [review]
Thank you for the addition @lhanusov, I added one comment

File: .github/scripts/pr-backport.sh (hunk 1 above)
Comment (mhajas) [reply .github/scripts/pr-backport.sh:88]: Thank you @jonkoops, I do not usually use the local `main` branch. This code assumes that everyone is using the `main` branch which may not be true. For that reason, it sounds safer to checkout to the previous branch.

Comment (mhajas) [reply .github/scripts/pr-backport.sh:88]: Locally, I always create a new branch when I am working on something using the following script: https://github.com/mhajas/.files/blob/main/work/bin/kc-git-checkout-new-branch

And when I want to try something without an issue I use Create a new branch from `upstream/main` in IntelliJ or checkout `upstream/main` which checks out the commit without creating a new branch.

Comment (lhanusov) [reply .github/scripts/pr-backport.sh]: adjusted, based on your proposal

**Review by mhajas — APPROVED** [review]
Thanks @lhanusov!

**Pull request merged**